### PR TITLE
Tagging changes.

### PR DESCRIPTION
**Pull request by SkyIsTheLimit**

### CODE REVIEW

File: client/src/components/Header/index.tsx [hunk 1]
```diff
@@ -1,77 +1,56 @@
...
 import Logo from '../Logo';
 import SortSelect from '../SortSelect';
 import TextSearch from '../TextSearch';
+import LandingHeader from '../LandingHeader';
 // import { Button } from 'react';
-import { useAppDispatch, useAppSelector } from 'hooks/useTypedRedux';
-import Button from 'components/Button';
-import useTracker from 'hooks/useTracker';
-import { Link, NavLink } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
```
Comment (codelastnight) [inline client/src/components/Header/index.tsx:11]: I think this is old code overriding the new one in this file

File: client/src/components/LandingFooter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+// @ts-nocheck
```
Comment (codelastnight) [inline client/src/components/LandingFooter/index.tsx:1]: same with all the LandingFooter files, this was moved in the `landing` repo a while back

File: fetch/src/post.ts [hunk 3]
```diff
@@ -1,7 +1,6 @@
 // @ts-nocheck
-
 const { Schema, model } = require('mongoose');
-
+import CustomTag from 'server/src/models/customtag';
```
Comment (steveand117) [inline fetch/src/post.ts:3]: This doesn't import properly on my machine

Comment (codelastnight) [reply fetch/src/post.ts]: fixed

File: client/src/components/Header/index.tsx (hunk 1 above)
Comment (codelastnight) [reply client/src/components/Header/index.tsx:11]: we can fix this during mergins

**Pull request merged**